app/backend/resources/schema.edn

```diff
@@ -1817,7 +1817,81 @@
           {:db/ident :land-owner-opinion/authority-position
            :db/valueType :db.type/string
            :db/cardinality :db.cardinality/one
-           :db/doc "Rich text containing the authoritys position on the opinion"}]]}]
+           :db/doc "Rich text containing the authoritys position on the opinion"}]]}
+
+ {:db/ident :migration/contracts
+  :txes [[{:db/ident :contract/number
+           :db/valueType :db.type/string
+           :db/cardinality :db.cardinality/one
+           :db/doc "Link to contract file in document management system for procurement part"}
+
+          {:db/ident :contract/procurement-number
+           :db/valueType :db.type/string
+           :db/cardinality :db.cardinality/one
+           :db/doc "Link to State procurement registry"}
+
+          {:db/ident :contract/thk-procurement-link
```

Comment:
Is this needed? Is this just generated based on the procurement id in THK?

> THK/THK- link to THK procurement  - not editable http://mnt-thk.mts.local/web/#/procurementobject/<proc_id>,

Reply:
Removed.